<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeadOR</class>
 <widget class="qMRMLWidget" name="LeadOR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>1067</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="arcSettingsCollapsibleButton">
     <property name="text">
      <string>Planning</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Transform Node: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="trajectoryTransformComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLinearTransformNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton_4">
     <property name="text">
      <string>Distance To Target</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_8">
      <item row="1" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Value: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLLinearTransformSlider" name="distanceToTargetSlider">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
        <property name="TypeOfTransform">
         <enum>qMRMLLinearTransformSlider::TRANSLATION_IS</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Transform Node: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="distanceToTargetComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLinearTransformNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="MECollapsibleButton">
     <property name="text">
      <string>Trajectories</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="layoutToggleFrame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="MEModelVisCheckBox">
           <property name="text">
            <string>Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="MELineVisCheckBox">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="METipVisCheckBox">
           <property name="text">
            <string>Tip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="METraceVisCheckBox">
           <property name="text">
            <string>NRMS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="MEClustersCheckBox">
           <property name="text">
            <string>Clusters</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="microElectrodeLayoutFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="2">
          <widget class="QToolButton" name="METoolButton_3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="leftMELabel">
           <property name="text">
            <string>left label</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QToolButton" name="METoolButton_6">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QToolButton" name="METoolButton_1">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="4">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Posterior</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="METoolButton_0">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QToolButton" name="METoolButton_4">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="4">
          <widget class="QLabel" name="label_3">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Anterior</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QToolButton" name="METoolButton_7">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QToolButton" name="METoolButton_2">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QToolButton" name="METoolButton_5">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QToolButton" name="METoolButton_8">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton { 
	background-color: rgb(253, 253, 253);
	border-radius: 15px; 
	border-color: rgb(182, 182, 182); 
	border-width: 1px; 
	border-style: solid; 
}
QToolButton:checked {
	background-color: rgb(0, 255, 255);
}
QToolButton::menu-indicator {
	image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QLabel" name="rightMELabel">
           <property name="text">
            <string>right label</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="layoutToggleFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="MECenterLayoutPushButton">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MEXLayoutPushButton">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MEPlusLayoutPushButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="stimulationCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Stimulation</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Active: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="stimulationActiveCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amplitud (mA): </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="stimulationAmplitudeSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>8.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Radius (mm): </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="amplitudeRadiusLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLLinearTransformSlider</class>
   <extends>qMRMLSliderWidget</extends>
   <header>qMRMLLinearTransformSlider.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>LeadOR</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>distanceToTargetSlider</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LeadOR</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>distanceToTargetComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LeadOR</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>trajectoryTransformComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
